aztec_gddt/logic.py

```diff
@@ -351,6 +351,8 @@ def p_commit_bond(params: AztecModelParams,
     advance_blocks = 0
     transfers: list[Transfer] = []
 
+    MAGIC_BOND_AMOUNT_CONSTANT = 1.0  # HACK: Setting some non-zero bond amount.  
```

Comment:
this variable could be set to params['commit_bound_amount']

We would then need to add that key to the model parameter definitions, which is found at `types.py`, `params.py` and maybe at `experiments.py`.

Once this is done, this PR will be ready